src/test/java/gov/loc/repository/bagit/verify/BagVerifierTest.java

```diff
@@ -194,9 +199,14 @@ public void testErrorWhenMissingPayloadDirectory() throws Exception{
   public void testErrorWhenMissingPayloadManifest() throws Exception{
     copyBagToTestFolder();
     Bag bag = reader.read(Paths.get(folder.getRoot().toURI()));
-    File manifestFile = new File(folder.getRoot(), "manifest-md5.txt");
-    manifestFile.delete();
-    
+    File manifestFile = new File(folder.getRoot(), "manifest-md5.txt");  
+    if (PathUtils.isWindows()){    	
+    	Path manifestPath = FileSystems.getDefault().getPath(manifestFile.getAbsolutePath());	
+    	Files.move(manifestPath, manifestPath.resolveSibling("renamedManifext"));
+    }
+    else {
+    	manifestFile.delete();
```

Comment:
yeah I investigated it, and it turns out that on windows the bag reader wasn't letting go of the file so it wouldn't actually delete it. However adding https://github.com/LibraryOfCongress/bagit-java/blob/master/src/main/java/gov/loc/repository/bagit/reader/BagReader.java#L220 fixes the problem and the files deletes normally.